tests/ctst/world/Zenko.ts

```diff
@@ -134,47 +218,49 @@ export default class Zenko extends World {
      * @returns {undefined}
      */
     async setupEntity(entityType: string): Promise<void> {
-        const keycloakPassword = this.parameters.KeycloakPassword || '123';
-        const savedParameters = JSON.parse(JSON.stringify(this.cliOptions));
+        const saved: Saved = this.getSaved() as Saved;
+        const usedParameters = this.parameters as Parameters;
+        const keycloakPassword = usedParameters.KeycloakTestPassword as string || '123';
+        const savedParameters = JSON.parse(JSON.stringify(this.cliOptions)) as object;
         this.resetGlobalType();
         switch (entityType) {
-            case EntityType.ACCOUNT:
-                await this.prepareRootUser();
-                this.saved.type = EntityType.ACCOUNT;
-                break;
-            case EntityType.IAM_USER:
-                await this.prepareIamUser();
-                this.saved.type = EntityType.IAM_USER;
-                break;
-            case EntityType.STORAGE_MANAGER:
-                await this.prepareARWWI(
-                    this.parameters.StorageManagerUsername || 'storage_manager',
+        case EntityType.ACCOUNT:
+            await this.prepareRootUser();
+            saved.type = EntityType.ACCOUNT;
+            break;
+        case EntityType.IAM_USER:
+            await this.prepareIamUser();
+            saved.type = EntityType.IAM_USER;
+            break;
+        case EntityType.STORAGE_MANAGER:
+            await this.prepareARWWI(
+                    usedParameters.StorageManagerUsername as string || 'storage_manager',
```

Comment:
Since changing to Records above, this has changed. Let me know if this is still an issue

Reply:
Yes, I still see force-casts in the logic for this object, which should be enforced, i.e. we want to know in advance all the parameters we support, with their type. This applies to all the changes related to `this.parameters`

Reply:
What I'm foreseeing here is another issue.
Let's take this example: `parameters.StorageManagerUsername`.
, the issue here is we are casting to a string.
So to fix this, we put :
```
interface ZenkoWorldParameters {
   StargeManagerUsername?: string;
}
```
The cast is still necessary to avoid TypeScript complaining that the variable may be undefined, as the linter doesn't like use of non-null assertion (`!` operator).
For strings this can be avoided by defaulting the parameter to empty string I suppose, but number values such as for ports, or booleans such as `ssl` I'm not so sure.
I feel like the purpose of these parameters are to allow for flexible inputting to the World object, so I somewhat disagree with the notion that we should be hard typing this, but I could be wrong.

In any case, after rubber ducking here and exploring the parameters we use, everything is pretty much a string apart from ssl, maybe just something like this will work:
```
interface ZenkoWorldParameters {
   [key: string]: string;
   ssl: boolean;
}
```
And any other variable that has a different type can be specified, what do you think?

Reply:
addressed by adding world [parameters](https://github.com/cucumber/cucumber-js/blob/main/docs/support_files/world.md#world-parameters-1) with cucumber